content/enable-hibernation.md

```diff
@@ -20,6 +20,10 @@ section: software
 tableOfContents: true
 ---
 
+## Disclaimer
+
+Hibernation is not officially supported on Pop!_OS as of 22.04. This documentation is supplied for your own explorative purposes. Any changes to your system configuration may break hibernation, or hibernation may not work at all. As well, any support or assitance may not be provided.
```

Comment:
> As well, any support or assitance may not be provided.

`assitance` is a typo.

Can the sentence be simplified to just `Support or assistance may not be provided.`?

Reply:
Thanks! Fixed